src/utils/getTagsForTitle.js

```diff
@@ -7,8 +7,9 @@ const getTagsForTitle = title => {
     .replace('.', '')
     .toLowerCase()
     .trim()
+  const titleWithoutSpaces = titleWithoutJs.replace(new RegExp(/\s/, 'gm'), '-')
```

Comment:
figured that titles like 'content design' could have the 'content-design' tag, so we might just well add it